BackEnd/src/routes/api/course.js

```diff
@@ -49,9 +49,9 @@ createRoutes([
             }
             // TODO: get all courses for user from DB
             return res.status(200).json( [
-                {courseId: '2019WI2', year: 2019, subjectId: 'WI', subjectName: 'Wirtschaft'},
-                {courseId: '2019EN1', year: 2019, subjectId: 'EN', subjectName: 'Englisch'},
-                {courseId: '2019DE1', year: 2019, subjectId: 'DE', subjectName: 'Deutsch'},
+                {courseId: 1, year: 2019, subjectId: 0, subjectName: 'Wirtschaft'},
```

Comment:
Hierfür gibts noch keine Datenbanktabelle?

Reply:
Was meinst du? Wird doch mit den Tabellen Courses und Subjects abgedeckt. Zumal das nichts mit diesem Pull-Request zu tun hat. Oder meinst du, ob dieses Feature noch nicht von der DB abgefragt wird? Nein, das hier sind nur Mockdaten. Die Abfrage wird in #58 umgesetzt.